docs/reference/migration/migrate_8_0/security.asciidoc

```diff
@@ -306,6 +306,38 @@ renamed to better reflect its intended use.
 *Impact* +
 Assign users with the `kibana_user` role to the `kibana_admin` role.
 Discontinue use of the `kibana_user` role.
+====
+
+[discrete]
+[[saml-realm-nameid-changes]]
+===== The default value of `nameid_format` setting has been removed.
+
+.The `nameid_format` SAML realm setting has no default value.
+[%collapsible]
+====
+*Details* +
+In SAML, Identity Providers (IdPs) either release a `NameID` or attempt to
+conform with the requirements of a Service Provider (SP). The SP declares its
+requirements in the `NameIDPolicy` of an authentication request. In {es}, the
+`nameid_format` SAML realm setting controls the `NameIDPolicy`.
+
+Previously, the default value for `nameid_format` was
+`urn:oasis:names:tc:SAML:2.0:nameid-format:transient`. This default created
+authentication requests that would require the IdP to release `NameID` with a
+transient format.
+The default value has now been removed. This means that {es} will be default
+create SAML Authentication Requests that do not put forward such requirements
+to the Identity Provider.
+
+If you want to retain the previous behavior, you can set `nameid_format`
+to `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`.
+
+*Impact* +
+To avoid issues, explicitly configure `nameid_format`. If you don't configure
```

Comment:
This is not what I am trying to say. If we instruct everyone to set this value, we might as well leave the default in place. What I am trying to say is that "this should probably be ok, but if you know you want the old behavior or you don't know how your IDP is configured but it was working so far and you don't care to try, set this setting to this value explicitly"